Formula/bacula-fd.rb

```diff
@@ -18,24 +18,22 @@ def install
     #   (conio support not tested)
     # * working directory in /var/lib/bacula, reasonable place that
     #   matches Debian's location.
-    readline = Formula["readline"].opt_prefix
     system "./configure", "--prefix=#{prefix}",
-                          "--sbindir=#{bin}",
```

Comment:
Sorry! I mean until we've decided on a resolution to that discussion: let's leave formulae that are using `bin`/`sbin` as-is.